usr/lib/common/key.c

```diff
@@ -331,6 +331,16 @@ CK_RV key_object_validate_attribute(TEMPLATE *tmpl, CK_ATTRIBUTE *attr,
          */
         TRACE_ERROR("%s\n", ock_err(ERR_ATTRIBUTE_READ_ONLY));
         return CKR_ATTRIBUTE_READ_ONLY;
+    case CKA_IBM_ATTRBOUND:
```

Comment:
You check the attribute here (great!), but you do not set the attribute in key_object_set_default_attributes(). I guess this is by intention? That way a key object won't have that attribute, unless the user specifies it explicitly at object creation. 
That's OK, but all the places where you try to get the value of CKA_IBM_ATTRBOUND should be able to handle the situation that the attribute is not there, and should probably assume FALSE as the default value.

Reply:
That is done intentionally.  Since CKA_IBM_ATTRBOUND should be EP11-specific I did not want to set it in all cases.  And yes, not having the attribute set all the time causes some ugly code when dealing with non-present attribute values, but the code (after an update to this PR) will handle this correctly.